src/components/search/searchresults.js

```diff
@@ -464,7 +464,6 @@ define(['layoutManager', 'globalize', 'require', 'events', 'connectionManager',
             showTitle: true,
             overlayText: false,
             centerText: true,
-            action: 'play'
```

Comment:
Add `overlayPlayButton: true` to add play button for mobile.
TV will still go to itemDetails, but this is OK for now.